keyboards/themadnoodle/udon13/info.json

```diff
@@ -0,0 +1,75 @@
+{
+    "manufacturer": "The Mad Noodle",
+    "keyboard_name": "Udon13",
+    "maintainer": "The-Mad-Noodle",
+    "url": "https://www.madnoodleprototypes.com/",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "oled": true
```

Comment:
If OLEDs are enabled at the keyboard level, there needs to be some code to display something on them at the keyboard level.